src/styles/index.scss

```diff
@@ -1,3 +1,97 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background-color: $basicWhite;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    height: 135px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 5px;
+    color: $mainAcsent;
+  }
+
+  &__code {
+    font-size: 10px;
+    align-self: start;
+    margin-bottom: 17px;
+    color: $secondaryColor;
+  }
+
+  &__rate {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    color: $mainAcsent;
```

Comment:
There seems to be a typo in the variable name `$mainAcsent`. It should likely be `$mainAccent` if you intended to use the word 'accent'. Please check the variable definition in your `variables.scss` file.